Source/monster.cpp

```diff
@@ -4503,20 +4502,20 @@ void MissToMonst(Missile &missile, Point position)
 	}
 }
 
-Monster *MonsterAtPosition(Point position)
+Monster *MonsterAtPosition(Point position, bool strict)

```

Comment:
`ignoreMovingMonsters` maybe? `-id` is used for the tile a monster is leaving (or moving into...) as well as charging monsters. Most of the time the caller still wants the tile the monster is moving into when they're walking normally though so there's still potential confusion there...

Reply:
Isn't the -id also used for monsters that occupy multiple tiles?

Reply:
> Isn't the -id also used for monsters that occupy multiple tiles?

are there any?

Reply:
only cows in town, and town uses a special case for hit detection in cursor.cpp anyway which could be called out as an exception

Reply:
Ah yes, cows 🐮. I remember seeing -id being used for that but I wasn't sure where 😄 

I agree that that's an exception so I will rename `strict` to `ignoreMovingMonsters`

Reply:
Paramter renamed